content/ko/docs/helm/helm_uninstall.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Helm Uninstall"
+---
+
+## helm uninstall
+
+릴리스 언인스톨
+
+### 개요
+
+
+이 명령어는 릴리스 이름을 사용하여 릴리스를 언인스톨한다.
+
+차트의 마지막 릴리스 및 릴리스 내역과 관련된 모든 리소스를 제거하여 
+나중에 사용할 수 있도록 해제한다.
+
+
+실제 언인스톨은 하지 않고 릴리스가 언인스톨되는 것을 확인해보려면
+'--dry-run' 플래그를 사용하자.
+
+
+```
+helm uninstall RELEASE_NAME [...] [flags]
+```
+
+### Options
+
+```
+      --description string   사용자 정의 설명 추가
+      --dry-run              언인스톨 시뮬레이션
+  -h, --help                 uninstall 에 대한 도움말
+      --keep-history         모든 관련 리소스를 제거하고 릴리스를 삭제로 표시하지만 릴리스 내역은 유지
+      --no-hooks             언인스톨 중 훅(hook) 실행 방지
+      --timeout duration     개별 쿠버네티스 작업(예: 훅(hook)을 위한 작업)을 기다리는 시간 (기본값 5m0s)
+```
+
+### 상위 명령에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 true이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  파일처럼 표준 오류로도 로그 출력
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 문자열 전달 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   이 값이 비어있지 않을 경우, 이 값에 지정된 디렉토리에 로그 파일 쓰기 수행
+      --log-file string                  이 값이 비어있지 않을 경우, 이 값에 지정된 파일명으로 로그 파일 쓰기 수행
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가 바이트이며 값이 0일 경우 최대 파일크기 제한 없음(기본 값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 : true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```

Comment:
Same question (which is, why is this different from the English default?)

Reply:
There seems to be a mistake. Because every helm command has that same option.
From now on, we should translate and review more carefully.

Reply:
I will modify every I PRed as your comment like issue #810 :D